plugin-infra/go-plugin-access/src/main/java/com/thoughtworks/go/plugin/access/authorization/AuthorizationExtension.java

```diff
@@ -224,6 +224,25 @@ public AuthenticationResponse onSuccess(String responseBody, Map<String, String>
         });
     }
 
+    public boolean doesUserExists(String pluginId, String username, SecurityAuthConfig authConfig) {
```

Comment:
Maybe `isValidUser` instead of `doesUserExists`. 
- The user account can be disabled by admin
- The user account can be temporarily locked

Reply:
This call should not be made for the plugin using extension version v1.

Reply:
@bdpiparva -- we are not making this call for `v1` extension. It should throw an `UnsupportedOperationException` if we try to make a call for `v1` extension.

I will rename the plugin-extension call to `is-valid-user` and change the relevant code too?